### DISCUSSION TITLE
159

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#105] [General]
# 潮流周刊第159期 - 自制咖啡

封面图来源于我装的咖啡豆，倒出来那一瞬间超级香，元旦一时兴起去买了一台半自动咖啡机，这几天每天 2 杯自制咖啡，还挺美味。

https://weekly.tw93.fun/posts/159-%E8%87%AA%E5%88%B6%E5%92%96%E5%95%A1

**Comment by GitWednesday** [comment]
Bio Link 的插图和 Github Worth 的重复了

**Reply by tw93** [reply]
抱歉 已经修复，我的图片cdn工具最近有些Bug，后面多检查一下

**Comment by GetToSet** [comment]
新年快乐🥳

pubDate 好像错了呢~

**Reply by tw93** [reply]
哈哈哈 真...  过年了

**Comment by iniFeng** [comment]
摘抄的内容真深刻啊

**Reply by tw93** [reply]
这本书还是很不错的
